server/build_event_protocol/config/config.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+import "flag"
+
+var tagsEnabled = flag.Bool("app.tags_enabled", false, "Enable setting tags on invocations via build_metadata")
```

Comment:
any reason you can't put this flag in the only package that it's used? we have way too many `config.go` files.

Reply:
Meh, I had it in my head that I'd use the same flag to control the UI, but honestly it's probably better to keep the ingestion separate from the display.